.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Description
+Briefly describe the problem or reason for the changes implemented in the PR
+
+# Trac ticket
+Link to the Trac ticket for this PR (all PRs except really trivial issues or 
+tests adding need an accepted ticket).
+
+# UI Changes
+Please attach screenshots in light and dark mode for UI changes.
+
+# Checklist
+- [ ] I have updated the `has patch` flag in the Trac ticket
```

Comment:
Use quotes rather than code, and say “checked” for clarity. Also turn into a sentence.
```suggestion
- [ ] I have checked the “has patch” flag in the Trac ticket.
```